twisted/internet/udp.py

```diff
@@ -323,7 +323,7 @@ def writeSequence(self, seq, addr):
         @param addr: A tuple of (I{stringified IPv4 or IPv6 address},
             I{integer port number}); can be L{None} in connected mode.
         """
-        self.write("".join(seq), addr)
+        self.write(b"".join(seq), addr)
 
```

Comment:
@scop you are right.  I am going to take this part out and submit a separate ticket for that, and add a unit test for writeSequence().